Makefile

```diff
@@ -106,6 +106,16 @@ deps:
 	test -f $(BINPATH)/golangci-lint || curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(BINPATH) v1.21.0
 	test -f $(BINPATH)/shellcheck || curl -sfL "https://storage.googleapis.com/shellcheck/shellcheck-v0.4.7.linux.x86_64.tar.xz" | tar -xJv --strip-components=1 -C $(BINPATH)
 
+.PHONY: pre-commit-install
+pre-commit-install:
+	test -f $(BINPATH)/bin/pre-commit || curl -sfL https://pre-commit.com/install-local.py | HOME=$(BINPATH) python -
+	$(BINPATH)/bin/pre-commit install -t pre-commit -t commit-msg
+
+.PHONY: pre-commit-uninstall
+pre-commit-uninstall:
+	test -f $(BINPATH)/bin/pre-commit || curl -sfL https://pre-commit.com/install-local.py | HOME=$(BINPATH) python -
```

Comment:
Is it possible to pin to a specific version to avoid issues with new releases?

Reply:
For `install-local.py`, it does not support pass specific version when pip install.

Other installation ways are https://pre-commit.com/#install

Any better idea?

Reply:
it was mostly curiosity, not a big deal :sweat_smile: 
it's fine with me, but I defer to others the approval
Thanks for the improvement